data/ext/pending/issue-1423.ttl

```diff
@@ -30,3 +51,46 @@
     :source <https://github.com/schemaorg/schemaorg/issues/1423> ;
     rdfs:comment "Human-readable terms of service documentation." .
 
+:endpointUrl a rdf:Property ;
+    rdfs:label "endpointUrl" ;
+    :category "issue-1423" ;
+    :domainIncludes :WebAPI ;
+    :isPartOf <http://pending.schema.org> ;
+    :rangeIncludes :EntryPoint,
+        :URL ;
+    :source <https://github.com/schemaorg/schemaorg/issues/1423> ;
+    rdfs:comment "The root location or primary endpoint of the API." .
+
+:conformsTo a rdf:Property ;
+    rdfs:label "conformsTo" ;
+    :category "issue-1423" ;
+    :domainIncludes :WebAPI ;
+    :isPartOf <http://pending.schema.org> ;
+    :rangeIncludes :URL ;
+    :source <https://github.com/schemaorg/schemaorg/issues/1423> ;
+    rdfs:comment "The URL reference of an established standard to which the described API conforms, for example `https://jsonapi.org/format/1.0/`, `https://grpc.io/`, or `http://www.hydra-cg.com/spec/latest/core/`." .
```

Comment:
https://jsonapi.org/format/1.0/ is a good example
http://www.hydra-cg.com/spec/latest/core/ is  a vocabulary for describing WebAPIs, not a specification of an API. Not a good example.
https://grpc.io/ is about an RPC framework "gRPC is based around the idea of defining a service, specifying the methods that can be called remotely with their parameters and return types" Doesn't look like an API specification, but a framework for defining interface. Not a good example.

Here's some API definitions that should be better examples
https://mmisw.org/orrdoc/api/
https://www.ncdc.noaa.gov/paleo-search/api
http://www.opengis.net/doc/IS/wfs/2.0.2
http://service.iris.edu/irisws/timeseries/1/

Reply:
The idea with these examples is to show the breadth of `conformsTo`: it is not limited to just API specifications, but indeed any applicable standard or specification.

Reply:
Not very useful to point at a web page or specification that doesn't really tell you anything about the API.  I think we have a communication problem here.    Did you look at my examples?

Reply:
Perhaps it's worth noting that `conformsTo` is an array, so can include both specific and more generally applicable standards or specifications. 

Of your suggested examples, it looks like they cover both `endpointDescription` and `conformsTo` (noting [the distinction](https://www.w3.org/TR/vocab-dcat-2/#Property:data_service_endpoint_description) between these).

| `endpointDescription` | `conformsTo` |
|------------------------|---------------|
| https://mmisw.org/ontapi/ | https://mmisw.org/orrdoc/api/ |
| https://www.ncdc.noaa.gov/paleo-search/api | ? |
| ? | http://docs.opengeospatial.org/is/09-025r2/09-025r2.html |
| http://service.iris.edu/irisws/timeseries/1/ | ? |

In the first of these examples the link to http://sensorml.com/ontapi/ is broken, which leads me to question how up-to-date that document is if we were to reference it?

However, I see your point regarding gRPC, as arguable it fits more neatly within `apiTransport`. Hydra shapes both the design of an API and its documentation, however if controversial it can easily be removed.

Given the deadline for this release is tomorrow, I will remove gRPC and Hydra from the examples for now. We can always create a subsequent PR with more examples later on.

Reply:
CC @MikeRalphson

Reply:
Good point that the distinction between a specification and an endpoint description can be pretty fuzzy, especially in the case of the numerous APIs that are one-ofs.   I thought the logic of endpointDescription was that it should be a link to  machine readable (actionable) document like an OpenAPI or OGC getCapabilities instance (see https://github.com/webapi-discovery/rfcs/issues/13).  If there's a web page that describes the API for people, that's pretty much a specification, but it might better be considered so:documentation (see https://github.com/webapi-discovery/rfcs/issues/10).   In an ideal world someone defining an API that they think someone else might want to use or describe as a so:WebAPI, they should specify a URI (versioned!) to use in so:conformsTo.  This is wishful thinking at this point, so people are going to be guessing what to put in conformsTo or documentation....

let's see how it plays out in the wild when people start using it.

by the way, an endpoint description associated with http://docs.opengeospatial.org/is/09-025r2/09-025r2.html would be a URL for the OGC GetCapabilities instance associated with a service instance (e.g. https://geoservices.gelderland.nl/server/services/Inspire/INSPIRE_PS/MapServer/WFSServer?request=GetCapabilities&service=WFS)

Reply:
So just to spell it out, in this case 

conformsTo = http://docs.opengeospatial.org/is/09-025r2/09-025r2.html (actually http://www.opengis.net/doc/IS/wfs/2.0.2) 

endpointDescription = https://geoservices.gelderland.nl/server/services/Inspire/INSPIRE_PS/MapServer/WFSServer?request=GetCapabilities&service=WFS)

Reply:
Yes good catch, indeed I'd confused our own definitions of `endpointDescription` and `documentation` above.

The table should in fact read:

| `documentation` | `endpointDescription` | `conformsTo` |
|------------------|------------------------|---------------|
| https://mmisw.org/ontapi/ | https://mmisw.org/ontapi/swagger_mmiorr.yaml | https://mmisw.org/orrdoc/api/ |
| https://www.ncdc.noaa.gov/paleo-search/api | ? | ? |
| ? | https://geoservices.gelderland.nl/server/services/Inspire/INSPIRE_PS/MapServer/WFSServer?request=GetCapabilities&service=WFS | http://www.opengis.net/doc/IS/wfs/2.0.2 |
| http://service.iris.edu/irisws/timeseries/1/ | ? | ? |

I've clarified the definitions in this PR, to minimise such potential confusion: https://github.com/schemaorg/schemaorg/pull/2635/commits/33b33fe1c26f97becc98d7c982004693dcc42818

Thanks both!